src/core/common/instance.hpp

```diff
@@ -190,6 +190,14 @@ class Instance : public otInstance, private NonCopyable
      */
     void Reset(void);
 
+#if OPENTHREAD_RADIO
+    /**
+     * This method resets the internal states of the instance.
+     *
+     */
+    void StackReset(void);
```

Comment:
This function only reset the RCP stack rather than the all OpenThread stack. The function name confuses code readers. How about changing the name to `RcpReset` or something.

Reply:
We are planning to change it to a full stack reset in the future. But I'm OK with changing the name for now.

Reply:
Updated.